cr.yaml

```diff
@@ -0,0 +1,2 @@
+owner: abnamro
```

Comment:
Is it really necessary to use config file cr.yaml? Doesn't it automatically pick the owner and git repo name automatically? If it is not necessary we can avoid this config file. If we need it then I would give it some meaningful name like helm-chart-releaser-config.yaml.

Reply:
Also, the gh-pages branch currently is cloned of the main branch? I think we don't need to clone it from the main branch.
It is just going to be used to host the helm packages.
I looked at a few examples. Take a look here: https://github.com/search?q=helm%2Fchart-releaser-action%40v1.5.0&type=commits

Reply:
After doing some additional testing, the cr.yaml file is indeed unnecessary since the workflow operates finely without it. Removed this file in the latest commit.

For your other point, the gh-pages branch could indeed be emptied and only contain the needed index.yaml file and maybe a README to help those in need of assistance. What would your preferred course of action be?